giftless/error_handling.py

```diff
@@ -17,11 +18,22 @@ def init_app(self, app):
         for code in default_exceptions:
             app.errorhandler(code)(self.error_as_json)
 
+        # Specifically handle AccessDenied with a JSON response and 403 status
+        app.errorhandler(AccessDenied)(self.access_denied_as_json)
+
     @classmethod
     def error_as_json(cls, ex):
         """Handle errors by returning a JSON response
+        only maps HTTP-based exceptions from werkzeug.exceptions (like NotFound or Forbidden)
         """
         code = ex.code if hasattr(ex, 'code') else 500
         data = {"message": str(ex)}
+        logging.debug(f"Returning error response: {data} with status {code}")
```

Comment:
Tests are failing saying "logging is not found"

Reply:
Ah! That makes sense. Couldn't see this response even though I expected it.